lib/snmp/src/agent/snmpa.erl

```diff
@@ -2473,7 +2459,7 @@ log_to_txt(LogDir, Mibs, OutFile, LogName, LogFile, Start) ->
       LogName :: string(),
       LogFile :: string(),
       Block   :: boolean(),
-      Start   :: null | snmp:log_time();
+      Start   :: snmp:log_time();
```

Comment:
Why remove 'null'? Have you updated 'snmp:log_time()' type?

Reply:
good catch! I do not know why I seem to think it was included in `snmp:log_time/0`.
I have reverted all other places where I changed this. Thanks